app/models/proposal.rb

```diff
@@ -270,19 +264,22 @@ def create_new_observation(user, adder, reason)
     # invalidate relation cache so we reload on next access
     self.observers(true)
     # when explicitly adding an observer using the form in the Proposal page...
-    if adder
-      add_observation_comment(user, adder, reason) unless reason.blank?
+    if adder && reason
+      add_observation_comment(user, adder, reason)
```

Comment:
before it was `add_observation_comment(user, adder, reason) unless reason.blank?` -- so it would *not* be called if blank. 

When I remove the `&& reason` here, `spec/models/proposal_spec.rb:304` fails, so it does appear to be covered by tests

BTW, I am assuming you thought the behavior was different because you didn't see the trailing `unless` - this is why I am firmly anti-trailing conditionals!